src/main/ruby/core/method.rb

```diff
@@ -21,7 +21,9 @@ def inspect
   def curry(curried_arity = nil)
     self.to_proc.curry(curried_arity)
   end
-  
+
```

Comment:
We don't usually like to commit white-space changes unless you're fixing something egregious. Could I get you to un-stage this change, please?